modules/local/add_info_to_vcf/environment.yml

```diff
@@ -1,3 +1,4 @@
+name: gawk
```

Comment:
Isn't it a problem that all 3 local have the same environment name?

Reply:
It _should_ just use the same conda env for all of them (just like it uses the same image, but makes a new container each process)